### DISCUSSION TITLE
ESP32 Basic Synth i2S interface Compilation ERROR

### DISCUSSION THREAD

**Discussion by chrebner** [marcel-licence/esp32_basic_synth#106] [Q&A]
Hi Marcel 
i ´m trying to compile the "ESP32 basic synth project"  into a ESP32 DEVKIT module and i always have this message in the Arduino IDE:

 _exit status 1
Compilation error: i2s_interface.h: No such file or directory_

I have not found this library in the repositorie, could you please give me a hand?

Thank

**Comment by marcel-licence** [comment]
Hi,

thank you for your message.
I've restructured the project over the last days.
I've added the missing files 20 hours ago to the ML_SynthTools library.

It should now work. Please let me know if it works after an update.

**Reply by chrebner** [reply]
Hi,

excellent Marcel !! Now i could compile and upload all projects without errors en ESP32 (DEKIT and NodeMCU32S)

Thanks
